doc/api/toolkits/mplot3d/view_angles.rst

```diff
@@ -38,3 +38,156 @@ further documented in the `.mplot3d.axes3d.Axes3D.view_init` API.
 
 .. plot:: gallery/mplot3d/view_planes_3d.py
    :align: center
+
+
+.. _toolkit_mouse-rotation:
+
+Rotation with mouse
+===================
+
+3D plots can be reoriented by dragging the mouse.
+There are various ways to accomplish this; the style of mouse rotation
+can be specified by setting ``rcParams.axes3d.mouserotationstyle``, see
+:doc:`/users/explain/customizing`.
+
+Originally (prior to v3.10), the 2D mouse position corresponded directly
+to azimuth and elevation; this is also how it is done
+in `MATLAB <https://www.mathworks.com/help/matlab/ref/view.html>`_.
+To keep it this way, set ``mouserotationstyle: azel``.
+This approach works fine for polar plots, where the *z* axis is special;
```

Comment:
We have "polar plots" that are 2D, so it would be better if this were written to more specifically mean something in 3D only, if possible.

Reply:
Yes, 'polar plot' is sloppy, 'spherical coordinate plot' seems more appropriate, I'll change it.